src/internals/network/getDefaultNetworkHelpers.js

```diff
@@ -1,62 +1,72 @@
 const DEFAULT_NETWORK_HELPERS = {
   getToken: () => 'token',
-  requestGET() {
+  async requestGET() {
+    const token = await Promise.resolve(this.getToken && this.getToken())
```

Comment:
I don't understand the need for `Promise.resolve` here, wouldn't `const token = this.getToken && await this.getToken()` work the same?

Reply:
Hey, good catch I didn't realize that await would turn the value into a promise, I thought it only worked on promises, that's why I had the promise.resolve :D I'll update this